mobile/src/pages/Login.tsx

```diff
@@ -62,7 +62,7 @@ export default function Login({ navigation }) {
             disabled={isPending || invalidCount >= 5}
             onPress={() => {
               setIsPending(true);
-              login(username, password)
+              login(email.trim(), password)
```

Comment:
wait Ive got the feeling that if the api let you submit here or in password with an extra space shouldnt that trim be in web/scr/api?